test/fixtures/expression/binary/migrated_0018.js

```diff
@@ -0,0 +1,2 @@
+'\
```

Comment:
Let's have the file named something else than `migrated_0018` (such a pattern was due a migration from the old test suite).
I suggest `multiline_string` or something similar.